### DISCUSSION TITLE
Welcome to sushiksha-website Discussions!

### DISCUSSION THREAD

**Discussion by 18praneeth** [18praneeth/sushiksha-website#224] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Maaz-Code** [comment]
Is this repository taking part in this year's Hacktoberfest and if yes, then can you please correct the label's spelling to make it a valid repo. I guess 'k' is missing in the spelling. Thank you!
